coss/base/templates/base.html

```diff
@@ -90,6 +90,7 @@
 
     {% compress js %}
       {# Global javascript #}
+      <script type="text/javascript" src="{% static 'lib/jquery/dist/jquery.min.js' %}"></script>
```

Comment:
I am not sure that we need this here. Probably the `home` app is a better place.

Reply:
Most probably we'll use jquery on other apps too, so it's better to be part of the base template. This is what the apps templates are currently extended from.

Reply:
We are going to restructure the project soon. Probably, it's better to organize everything in apps with similar functionality. Eg `groups and gatherings`. In any case we can discuss this later. Let's leave it as it is for now.